modules/ROOT/pages/configure-service-uri-task.adoc

```diff
@@ -0,0 +1,19 @@
+= Configuring the Service URI
```

Comment:
Imperative

Reply:
OK, I think we need to revisit the guidance. I see this in the SG:
https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-gerundtitles

But, my understanding was that a simple task should be imperative, not gerund.

Reply:
You can use gerunds in the nav to be consistent and use imperative in the H1.

Reply:
OK! Thanks!